server/app.js

```diff
@@ -1,5 +1,87 @@
 const express = require('express');
+
 const app = express();
 
+const filePath = process.env.ZACH_STORAGE || 'data.json';
+
+const fs = require('fs');
+
+app.use(express.static('../client/build'));
+
+app.get('/api/tickets', (req, res) => {
+  const data = fs.readFileSync(filePath);
+  const tickets = JSON.parse(data);
+  if (req.query.searchText && req.query.searchText !== '') {
+    // eslint-disable-next-line max-len
+    const filteredArr = tickets.filter((ticket) => {
+      const lowerCaseTitle = ticket.title.toLowerCase();
+      const lowerCaseQuery = req.query.searchText.toLowerCase();
+      return lowerCaseTitle.indexOf(lowerCaseQuery) !== -1;
+    });
+    res.send(filteredArr);
+  } else {
+    res.send(tickets);
+  }
+});
+
+app.post('/api/tickets/:ticketId/done', (req, res) => {
```

Comment:
Join paths to a single path with a url param `/api/tickets/:ticketId/:done` instead of using two paths